### DISCUSSION TITLE
Membuat objek Time series dengan ts()

### DISCUSSION THREAD

**Discussion by sabrinaa311** [teamalgoritma/community#338] [Q&A] [answered]
Permisi kak, jika saya memiliki range waktu seperti di gambar, parameter `start` dan `end` nya harus saya isi dengan apa ya kak sehingga bisa dirun kodenya?
![Capstone project machine learning - RStudio 24_09_2022 00_09_30](https://user-images.githubusercontent.com/107026622/192016190-fe2b6842-c81d-49a7-b74e-198b83dd91c5.png)

**Comment by fafilia** [comment, marked as answer]
Dear @sabrinaa311 ,

Untuk parameter start dan end silahkan diisi dengan periode data sesuai dengan usecase yang ada. Perlu diperhatikan bahwa Ibu harus cek terlebih dahulu kelengkapan data, apakah ada yang terlongkap/skip atau tidak. Jika ada silahkan dilengkapi terlebih dahulu baik data tanggal ataupun jumlah visitornya. 

Jika ini mengacu ke case Capstone FNB, Anda bisa lakukan eksplorasi kapan toko buka dan tutup karena untuk pola hariannya selalu sama. Sebagai tambahan, Anda dapat memfilter data hanya pada saat toko buka secara operasional saja. Karena di dalam hal ini semua data akan diubah ke obejct time series, untuk parameter start dan end bisa dikosongi saja sehingga Anda hanya tinggal mengatur data dan frekuensi yang digunakan untuk pembuatan object ts sesuai dengan fungsi yang Anda gunakan.

Demikian yang dapat saya sampaikan, semoga bermanfaat.
